bin/install

```diff
@@ -60,6 +70,45 @@ install_nodejs() {
 }
 
 
+install_aliased_version() {
+  local version=$1
+  local version_query=$2
+  local install_path=$3
+
+  # install the true version and only symlink it to the alias
+  >&2 echo "Installing alias $version_query as $version"
+  asdf install "$(plugin_name)" "$version" \
```

Comment:
Please don't invoke asdf commands from inside a callback script. This could have the unintentional affect of creating an infinite loop. It would be better to invoke the `install_canon_version` function defined in this file directly.